python/README.md

```diff
@@ -4,8 +4,50 @@ This minimal library contains the minimal set of functions you need to
 generate a VAPID key set and get the headers you'll need to sign a
```

Comment:
Keeping it as well. There are a lot of apps that like .md more than .rst, and it's a simple script to convert. (In fact, it's part of my publish script `pandoc --from=markdown --to=rst --output README.rst README.md` ) I'm ok with dropping the .rst from the repo, but since it's part of the python packaging, I'd rather keep it.